src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -457,6 +461,8 @@ void FixedwingAttitudeControl::run()
 			float deltaT = (hrt_absolute_time() - last_run) / 1000000.0f;
 			last_run = hrt_absolute_time();
 
+			static bool _flag_control_attitude_enabled_last = true;
```

Comment:
You should move this into the class instead of using a static.

Reply:
done.